tools/test-framework/src/chain/cli/query.rs

```diff
@@ -96,3 +96,60 @@ fn yaml_to_json_value(value: yaml::Value) -> Result<json::Value, Error> {
 
     Ok(parsed)
 }
+
+pub fn query_cross_chain_query_list(
```

Comment:
This looks like two functionalities mixed into one function. There should be one query function to return the result from `simple_exec`, and another assert function that calls the function, and asserts that the result is as expected.